src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```diff
@@ -53,6 +53,20 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Trimmer defaults -->
     <PublishTrimmed Condition="'$(PublishTrimmed)' == ''">true</PublishTrimmed>
     <TrimMode Condition="'$(TrimMode)' == ''">link</TrimMode>
+    <TrimmerRemoveSymbols Condition="'$(TrimmerRemoveSymbols)' == ''">false</TrimmerRemoveSymbols>
+
+    <!-- Runtime feature defaults to trim unnecessary code -->
+    <EventSourceSupport Condition="'$(EventSourceSupport)' == ''">false</EventSourceSupport>
+    <UseSystemResourceKeys Condition="'$(UseSystemResourceKeys)' == ''">true</UseSystemResourceKeys>
+    <EnableUnsafeUTF7Encoding Condition="'$(EnableUnsafeUTF7Encoding)' == ''">false</EnableUnsafeUTF7Encoding>
+    <HttpActivityPropagationSupport Condition="'$(HttpActivityPropagationSupport)' == ''">false</HttpActivityPropagationSupport>
+
+    <!--
+      _DebuggerSupport captures the explicitly specified value for DebuggerSupport. We'll use this value to determine if we need to
+      include pdbs as part of the publish output.
+    -->
+    <_DebuggerSupport>$(DebuggerSupport)</_DebuggerSupport>
```

Comment:
Is this necessary? Why not just use `$(DebuggerSupport)` below in the Task? If someone is changing it in the meantime, they explicitly want that behavior - not the cached value here.

Reply:
Also - since you are doing this above the line below - you are not getting the default setting... So by default `_DebuggerSupport` will be empty - which means `true` (since the default is to support debugging).